docs/get_started.md

```diff
@@ -98,81 +100,64 @@ the creation of a new management pack project.
      PNG format and 256x256 pixels. An icon file can be added later by copying the icon to the root project directory and
      setting the value of the `"pak_icon"` key to the icon's file name in the `manifest.txt` file.
 
-8. `Select a template for your project`
+8. `Select a language for the adapter:`
 
-     Both of the available options will generate a project structure that can be modified into an adapter:
+    The language the adapter uses determines the initial project structure.
+    Along with additional language-specific configuration.
+    Currently, `mp-init` offers the following languages:
 
-     `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
-      The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+    `Python`
+ 
+    `Java` 
 
-     `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
-      connection, collection, adapter definition, and endpoints logic.
+9. `(For Java projects only) Enter package name: `
 
-     For the purposes of this *Get Started* guide, select **Sample Adapter**.
+    The package name will be used to set up the package used by the adapter and the directory structure of the project.
+   
+    ???+ example 
+ 
+        ``` java
+        package com.mycompany;
+        ```
+
+10. `Select a template for your project`
+
+    Both of the available options will generate a project structure that can be modified into an adapter:
+
+    `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
+    The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+
+    `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
+    connection, collection, adapter definition, and endpoints logic.
+
+    For the purposes of this *Get Started* guide, select **Sample Adapter**.
 
 ???+ info
 
      For complete documentation of the `mp-init` tool, including an overview of its output,
-     see the [MP Initialization Tool Documentation](references/mp-init.md).
+     see the [MP Initialization Tool Documentation](references/mp-init.md). 
+     For Java projects documentation consult [Java project](references/python_project/index.md) section   
+     For Python projects documentation consult [Python project](references/java_project/index.md) section   
```

Comment:
I'm thinking these links may be better at the conclusion of the walkthrough. Also the wording is a bit awkward.